sci-libs/pagmo/pagmo-9999.ebuild

```diff
@@ -9,9 +9,9 @@ PYTHON_COMPAT=( python2_7 )
 inherit cmake-utils git-r3 python-single-r1
 
 DESCRIPTION="Parallelization engine for optimization problems"
-HOMEPAGE="http://pagmo.sourceforge.net/"
+HOMEPAGE="https://github.com/esa/pagmo"
 SRC_URI=""
-EGIT_REPO_URI="git://pagmo.git.sourceforge.net/gitroot/pagmo/pagmo"
+EGIT_REPO_URI="https://github.com/esa/${PN}.git git://git@github.com:esa/${PN}.git"
```

Comment:
Just checked https://github.com/gentoo-science/sci/pull/327. In the 2nd comment @junghans suggested to add git:// too. Can't remember why.

Further, you don't need an ssh key to pull from github via ssh, depending on the repository settings. For example `git clone git://github.com/DamnWidget/sublime-text.git` works without problems (layman -a added this to my */etc/portage/repos.conf/layman.conf*).

But sure, we can drop it. I would than grep over the whole overlay and fix that.